lib/src/notification/notification.dart

```diff
@@ -42,20 +42,26 @@ class BottomSlideNotification extends StatelessWidget {
 class SlideDismissible extends StatelessWidget {
   final Widget child;
 
-  final bool enable;
+  final DismissDirection direction;
 
-  const SlideDismissible({
-    required Key key,
+  SlideDismissible({
+    Key? key,
     required this.child,
-    required this.enable,
-  }) : super(key: key);
+    @Deprecated("use directions instead.") bool enable = true,
+    DismissDirection? direction = null,
+  })  : this.direction = direction != null
+            ? direction
+            : enable
+                ? DismissDirection.horizontal
+                : DismissDirection.none,
```

Comment:
According to [the documentation](https://api.flutter.dev/flutter/widgets/DismissDirection-class.html) DismissDirection.none is not valid.

Reply:
this documentation seems belong to the lastest stable version.

In fact, I currently plan to only support the flutter dev channel. Because this project was migrated to null safety, and null safety may not be compat with the stable branch of the old version (include 1.22.x).

Reply:
I realised that. :)
The problem is that I've updated my flutter and after that it complained about the package. But it could be that my flutter install is corrupted.

I'm going to reinstall it maybe it solves the problem (by the way I'm using the dev channel too).

Reply:
Yes, somehow my instal was corrupted. The dev branch somehow was mixed with tha master branch, or soehow it stayed in an older commit.

By the way, great package, keep up the good work!